### PR TITLE
Created documentation for the new satellite data structure

### PR DESCRIPTION
**Pull request by eltsai**
Created new documentation based on changes in the [public bucket](https://censoredplanet.org/data/raw)

### CODE REVIEW

**Review by ramakrishnansr — CHANGES_REQUESTED** [review]
I would also move this to the top of the documentation instead of the bottom.

File: docs/dns.rst [hunk 1]
```diff
@@ -732,6 +732,102 @@ Verify
...
 
+==============
+Satellite-v2.2-raw
+==============
+
+To provide raw data for easy data analysis, we made the following changes:
+
+1. Split data based on the country of resolvers so that it is easier to select and download data according to your country of interest.
+
+2. Separated the data collection phase and data analysis phase. Right now the Satellite data from our `public data bucket <https://data.censoredplanet.org/raw>`_ is truthful to the data collected without further analysis. We deprecated the “anomaly” field since there are misunderstandings that anomaly represents censorship.
```
[the remark this answers is not in the document]
Comment (eltsai) [reply docs/dns.rst]: Updated

**Review by ramakrishnansr — CHANGES_REQUESTED** [review]
I think moving the newer data format to the top of the page is a good idea? Also, for the old schema, can we add `deprecated`?

**Comment by eltsai** [comment]
Moved the new schema to the top and add `(deprecated)` after v1 and v2's schema

**Review by ramakrishnansr — APPROVED** [review]

**Pull request merged**